src/test/java/apicurito/tests/TestRunner.java

```diff
@@ -25,7 +26,11 @@
     public static void beforeTests() {
 
         if (OpenShiftUtils.xtf().getProject(TestConfiguration.openShiftNamespace()) == null) {
-            OpenShiftUtils.xtf().createProjectRequest(TestConfiguration.openShiftNamespace());
+            log.info("Creating new project " + TestConfiguration.openShiftNamespace());
```

Comment:
What's the reason behind abandoning the `createProjectRequest` method?

Reply:
ocp 4.1
createProjectRequest not working with ocp 4.1 right now.

Reply:
Cool, didn't know that, thanks!